lib/post_action_creator.rb

```diff
@@ -380,6 +380,8 @@ def create_message_creator
             .where("category_moderation_groups.category_id": @post.topic.category.id)
             .pluck(:name),
         )
+
+        create_args[:target_group_names].uniq!
```

Comment:
Minor code golfing but we can just make `create_args[:target_group_names]` a Set instead.